### PR TITLE
Add multicloud backingstore tests

### PR DESCRIPTION
**Pull request by MeridianExplorer**
Add tests that create and delete all possible backingstores via CLI and YAML on a given cluster.

### CODE REVIEW

**Review by fbalak — DISMISSED** [review]
Are backingstore deletion tests needed? Wouldn't it be tested if there was a check in [backingstore factory teardown](https://github.com/red-hat-storage/ocs-ci/blob/c4d5cd540436e09ee31dd809b24d2d0d57ef6f7d/ocs_ci/ocs/resources/backingstore.py#L146) for proper deletion of backingstore?

**Comment by MeridianExplorer** [comment]
@fbalak - 
First of all, thanks for the review!

1. I thought the same when I originally created the bucket creation tests. However, @ebenahar said that it's probably better to have a dedicated test that shows the success/failure of a specific component, instead of seeing it as a failure in the teardown of another.
Thus, we opted to have dedicated bucket deletion tests, and I followed suit when writing the backingstore tests :)

2. I added deletion verification independently of which path we'll choose to proceed with

**Review by fbalak — DISMISSED** [review]

File: tests/manage/mcg/test_multicloud.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework.testlib import MCGTest
+
+logger = logging.getLogger(__name__)
+
+
+class TestMultiRegion(MCGTest):
```
Comment (ebenahar) [inline tests/manage/mcg/test_multicloud.py:11]: Should this class be called `TestMultiCloud`? We already have such class in tests/manage/mcg/test_multi_region.py so I wasn't sure

Comment (MeridianExplorer) [reply tests/manage/mcg/test_multicloud.py:11]: Whoops. You're right. That's what I get for copy-pasting entire files.
Thanks :)

File: ocs_ci/ocs/resources/backingstore.py [hunk 2]
```diff
@@ -197,13 +244,29 @@ def _create_backingstore(method, uls_dict):
                             cmdMap[method.lower()][cloud.lower()](
                                 cld_mgr, backingstore_name, uls_name, region
                             )
-                        # Todo: Raise an exception in case the BS wasn't created
+                        wait_for_resource_state(
+                            OCP(
+                                kind="backingstore",
+                                namespace=config.ENV_DATA["cluster_namespace"],
+                            ),
+                            constants.STATUS_READY,
```
Comment (bernerhat) [inline ocs_ci/ocs/resources/backingstore.py:248]: there are two healthy statuses for backing store, I've created this constant `HEALTHY_PV_BS` for the pv backing store but maybe we should change its name to `HEALTHY_BS` and use it here

File: tests/manage/mcg/test_multicloud.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        ids=[
+            "AWS-CLI-1",
+            "AWS-OC-1",
+            "AZURE-CLI-1",
+            "AZURE-OC-1",
+            "GCP-CLI-1",
+            "GCP-OC-1",
+        ],
+    )
+    def test_multiregion_backingstore_creation(
```
Comment (bernerhat) [inline tests/manage/mcg/test_multicloud.py:38]: would change the test name to `test_multicloud_backingstore_creation`

File: tests/manage/mcg/test_multicloud.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        ids=[
+            "AWS-CLI-1",
+            "AWS-OC-1",
+            "AZURE-CLI-1",
+            "AZURE-OC-1",
+            "GCP-CLI-1",
+            "GCP-OC-1",
+        ],
+    )
+    def test_multiregion_backingstore_deletion(
```
Comment (bernerhat) [inline tests/manage/mcg/test_multicloud.py:69]: same as previous comment

File: ocs_ci/utility/utils.py [hunk 5]
```diff
@@ -1561,6 +1561,7 @@ def ceph_health_check_base(namespace=None):
         boolean: True if HEALTH_OK
 
     """
+    return True
```
Comment (bernerhat) [inline ocs_ci/utility/utils.py:1564]: Please remove

**Review by bernerhat — APPROVED** [review]
Looks Great!

**Review by bernerhat — APPROVED** [review]

**Review by fbalak — DISMISSED** [review]

**Review by fbalak — APPROVED** [review]

**Review by bernerhat — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**